kolibri/plugins/learn/assets/src/views/AlsoInThis.vue

```diff
@@ -148,18 +148,31 @@
         type: Array,
         required: true,
         default: () => [],
+        /** Content node with the following properties: id, is_leaf, title */
```

Comment:
I think we could even remove comments like this. Nice thing about `validateObject` is that it is self-documenting.